### PR TITLE
Design the structure of the calculator

### PR DESCRIPTION
**Pull request by principlebrothers**
### Design the structure of the calculator

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <div className="gOpsBody">
+            <GeneralOperations />
+          </div>
+          <div className="numBody">
+            <Numbers />
+          </div>
+          <div className="sideOps">
+            <Operations />
+          </div>
+        </section>
```
Comment (TedLivist) [inline src/components/Calculator.js:63]: - [x] We are expected to stick to the design. There seems to be a disparity in the placement of your numbers and the operations. They should be on the same line as we have in the design.

  Given design | Your design
  :---:|:---:
  design-1 | design-2

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @principlebrothers,

Good job so far!
✔️ Good documentation
✔️ Use of class-based components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (principlebrothers) [reply src/components/Calculator.js:63]: @TedLivist Thanks for the feedback. I have fixed that now.

**Review by julie-ify — APPROVED** [review]
Hi @principlebrothers, 

## Highlights!
- You sticked to the design guidline ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**